### DISCUSSION TITLE
Linux has some odd "jitter"

### DISCUSSION THREAD

**Discussion by ghost** [alvr-org/ALVR#1239] [Q&A] [answered]
i don't know if this has anything to do with async reprojection, but Linux ALVR feels a bit disorientating compared to the standard quest link.

**Comment by zmerp** [comment]
What about ALVR on Windows? How does it compare to ALVR on Linux or Oculus Link?

**Comment by ghost** [comment]
i'll test once i get my windows machine online.

**Comment by ghost** [comment, marked as answer]
v19 seems to have fixed it.
